app/src/debug/java/org/simple/clinic/playground/AwaitActivity.kt

```diff
@@ -0,0 +1,44 @@
+package org.simple.clinic.playground
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.Disposable
+import kotlinx.android.synthetic.debug.await_activity.*
+import org.simple.clinic.R
+import org.simple.clinic.await.Await
+import org.simple.clinic.await.Checkpoint
+import org.simple.clinic.util.unsafeLazy
+
+class AwaitActivity : AppCompatActivity() {
+  private lateinit var awaitDisposable: Disposable
```

Comment:
Instead of just `lateinit var Disposable` let's use `val CompositeDisposable`? That way we do not have to check for `isInitialized` in `onStop`.

Reply:
Done